### PR TITLE
feat: add vue macros

### PR DESCRIPTION
**Pull request by sxzz**
### Description

Add [Vue Macros](https://vue-macros.sxzz.moe/)

### CODE REVIEW

File: src/components/Input.vue [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    type="text"
+    v-bind="$attrs"
+    p="x-4 y-2"
+    w="250px"
+    text="center"
+    bg="transparent"
+    border="~ rounded gray-200 dark:gray-700"
+    outline="none active:none"
+  >
+</template>
```
Comment (userquin) [inline src/components/Input.vue:20]: is this component necessary? 
- why no just add defineModel on index page? or
- move attrs to index page: you're binding any attr to the component

Comment (sxzz) [reply src/components/Input.vue:20]: - v-model requires two SFC files, parent and child. So I have to create a child component to demonstrate that.
- It's just an example of extracting input style to an SFC.

Comment (userquin) [reply src/components/Input.vue:20]: ok

**Comment by antfu** [comment]
We should introduce it a bit in readme

**Comment by sxzz** [comment]
Added.

**Comment by sxzz** [comment]
ping @antfu

**Pull request merged**